setup.cfg

```diff
@@ -46,6 +47,7 @@ install_requires =
     pytz>=2021.3
     requests>=2.26.0
     rp2>=1.7.1
+    urllib3<2 # 2+ is incompatible with Python ~3.9
```

Comment:
Let's instead support Python 3.10...3.13.

Reply:
OK, should be good to go now. Thanks.